Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py

```diff
@@ -16,6 +16,7 @@
 
 ''' CONSTANTS '''
 
+SEARCH_RESULT_RETRIES = 10
```

Comment:
Do you know, is there any delay between each search or it just do immediate call? What do you think about adding backoff_factor to http client?

Reply:
Added